### PR TITLE
Ispn 7160/ispn 7158/spring test failures

### PR DESCRIPTION
**Pull request by slaskawi**
https://issues.jboss.org/browse/ISPN-7160
https://issues.jboss.org/browse/ISPN-7158

Good to be back :D

### CODE REVIEW

File: spring/spring4/spring4-common/src/test/java/org/infinispan/spring/session/InfinispanApplicationPublishedBridgeTCK.java [hunk 1]
```diff
@@ -64,6 +65,11 @@ public void testEventBridge() throws Exception {
 //      EventsWaiter.assertNumberOfEvents(() -> eventsCollector.getEvents(), SessionDestroyedEvent.class, 2, 10, TimeUnit.SECONDS);
    }
 
+   private void sleepOneSecond() {
+      long oneSecondSleep = System.currentTimeMillis() + 1000;
+      eventually(() -> System.currentTimeMillis() > oneSecondSleep);
```
Comment (danberindei) [inline spring/spring4/spring4-common/src/test/java/org/infinispan/spring/session/InfinispanApplicationPublishedBridgeTCK.java]: How is this better than a regular `sleep`?

Comment (danberindei) [inline spring/spring4/spring4-common/src/test/java/org/infinispan/spring/session/InfinispanApplicationPublishedBridgeTCK.java]: Oops, I should have read the bug description :)

Still, wouldn't it be possible to use a `TimeService`?

Comment (slaskawi) [inline spring/spring4/spring4-common/src/test/java/org/infinispan/spring/session/InfinispanApplicationPublishedBridgeTCK.java]: I tried to look into `TimeService` as you suggested but I couldn't find any nice way to hook in.

I could construct a loop using `isTimeExpired` but the end result would be the same. But maybe I missed something...

**Comment by danberindei** [comment]
Integrated, thanks Sebastian!

**Pull request closed without merging**